src/workos.ts

```diff
@@ -28,7 +28,7 @@ import { AuditLogs } from './audit-logs/audit-logs';
 import { Users } from './users/users';
 import { BadRequestException } from './common/exceptions/bad-request.exception';
 
-const VERSION = '3.9.1';
+const VERSION = '3.9.2';
```

Comment:
3?
this PR is on 5
https://github.com/workos/workos-node/pull/938/files

wouldn't that capture your changes

Reply:
This is for backwards compatibility for those users using v3.9.1